### PR TITLE
Added feature for updating transactions

### LINKED ISSUE #46: Feature with respect to updating transactions
It would better if there was a option to edit the transactions which are already saved.
Would like to enhance this feature

### PR DESCRIPTION
**Pull request by Villain45**
closes #46

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+.env
```
Comment (manuelalferez) [inline .gitignore:36]: Next.js uses the `.env.local` file by default to store environment variables. You can save your variables there and then remove this file: [Next.js Environment Variables Documentation](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables).

Comment (Villain45) [reply .gitignore]: Done...

Comment (manuelalferez) [reply .gitignore]: Amazing

**Comment by manuelalferez** [comment]
Looks good to go! Amazing job @Villain45. It works perfectly. Love it!

**Pull request merged**